docs/builtins/Commitments/validate-keypair.md

```diff
@@ -1,32 +1,33 @@
 ## validate-keypair
-The `validate-keypair` function enforces that the Curve25519 keypair composed of the public key (PUBLIC) and the private key (SECRET) provided as base-16 strings of length 32 match each other.
+
+Use `validate-keypair` to verify that the Curve25519 `public` key and `secret` key you specify are base-16 strings of 32 characters match each other.
 
 ### Basic syntax
 
-To validate a Curve25519 keypair, use the following syntax:
+To validate a Curve25519public and secret key pair, use the following syntax:
```

Comment:
```suggestion
To validate a Curve25519 public and secret key pair, use the following syntax:
```

Reply:
should we go with secret key? Most of the time the pair is called public and private key.

Reply:
I found both used throughout the docs. I started leaning towards secret because that seemed more consistent with `pact genkey` cli and  `kadena key` cli.